.github/workflows/php-test.yml

```diff
@@ -59,9 +59,9 @@ jobs:
         wordpress: [ 'latest' ]
         include:
           - php: '8.0'
-            wordpress: '6.0'
-          - php: '8.0'
-            wordpress: '6.1'
+            wordpress: '6.3'
+          - php: '8.1'
+            wordpress: '6.3'
```

Comment:
Right now, these two combinations are already covered by 8.1+latest and 8.0+latest above

We can remove them now and re-add once 6.4 is latest

Reply:
Thanks. Removed in https://github.com/WordPress/plugin-check/pull/292/commits/6ce9148503555293d46c4874c59d2c5049683e60